src/components/users/UserContacts.tsx

```diff
@@ -25,22 +32,29 @@ const StyledLink = styled(Link)`
     font-size: 14px;
 `;
 
+const StyledText = styled(Text)`
+    color: ${gray10};
+    margin-left: ${gapS};
+    font-size: 14px;
+`;
+
 const StyledCard = styled.div`
     height: 100%;
 `;
 
 type UserContactsProps = {
     user: User | undefined;
+    visible?: boolean;
```

Comment:
What's the purpose of this prop? To hide devices? Why is it needed?

Reply:
Because the devices aren`t used in the popup

Reply:
`hideDevices` would be a better suited name for a prop. Or maybe `popupMode`